sql/dj_setup_database.in

```diff
@@ -313,6 +318,8 @@ bare-install|install)
 	DBUSER=$domjudge_DBUSER PASSWD=$domjudge_PASSWD symfony_console domjudge:load-default-data
 	if [ "$1" = "install" ]; then
 		DBUSER=$domjudge_DBUSER PASSWD=$domjudge_PASSWD symfony_console domjudge:load-example-data
+		"$EXAMPLEPROBDIR"/generate-contest-yaml
+		( cd "$EXAMPLEPROBDIR" && yes y | "$BINDIR"/import-contest "${BASEURL}api" )
```

Comment:
Does this actually work for a clean install? I.e. how does it know the use admin:$pass in that case?

Reply:
The integration test is a clean install and it works :-)

It works via the `~/.netrc` as usual for the `import-contest` script.